ProjectReunion.sln

```diff
@@ -729,6 +737,54 @@ Global
 		{A3FBA80D-5B35-471F-9A45-DB4B29E195B9}.Release|x86.ActiveCfg = Release|x86
 		{A3FBA80D-5B35-471F-9A45-DB4B29E195B9}.Release|x86.Build.0 = Release|x86
 		{A3FBA80D-5B35-471F-9A45-DB4B29E195B9}.Release|x86.Deploy.0 = Release|x86
+		{4995CCDC-9B83-4A9A-9724-604B09E6AF00}.Debug_test|Any CPU.ActiveCfg = Debug|x64
```

Comment:
>Debug_test [](http://example.com/codeflow?start=41&length=10)

remove

Reply:
What is `Debug_test` for anyway? Looks like everything's got them but we never use them (that I know of). Delete them all?

Reply:
uhh, i thought you added that. But it's existing. It does change IsTDPConfiguration to true, although nobody seems to use IsTDPConfiguration. I guess it's ok to leave it alone.

Reply:
Any idea what `Debug_test` is for? I never saw it before last year in ProjectReunion.sln.

I added the project via VS on a PC which I think doesn't have TDP installed, if that matters

Reply:
related to TAEF testing configurations?  if it looks like cruft, I'd drop it

Reply:
Drop all of them or just these additions?